package.json

```diff
@@ -90,77 +101,87 @@
+    "react-draggable": "4.5.0",
+    "react-h5-audio-player": "3.10.1",
+    "react-intersection-observer": "10.0.0",
+    "react-redux": "9.2.0",
+    "react-toastify": "10.0.0",
+    "react-use": "17.6.0",
+    "react-virtualized": "9.22.6",
+    "read-last-lines": "1.8.0",
+    "redux": "5.0.1",
+    "redux-persist": "6.0.0",
+    "redux-promise-middleware": "6.2.0",
+    "rimraf": "6.1.2",
     "sanitize.css": "^12.0.1",
     "semver": "^7.7.1",
-    "sharp": "https://github.com/session-foundation/sharp/releases/download/v0.34.5/sharp-0.34.5.tgz",
-    "styled-components": "^6.1.15",
+    "sharp": "^0.34.4",
+    "styled-components": "6.1.19",
     "uuid": "11.1.0",
     "webrtc-adapter": "^4.2.2",
-    "zod": "^3.24.2"
+    "zod": "4.1.13"
   },
   "devDependencies": {
+    "@babel/cli": "7.28.3",
+    "@babel/core": "7.28.5",
+    "@babel/preset-env": "7.28.5",
     "@commitlint/cli": "^19.8.0",
     "@commitlint/config-conventional": "^19.8.0",
-    "@testing-library/jest-dom": "^6.4.6",
-    "@testing-library/react": "^15.0.7",
-    "@testing-library/user-event": "^14.6.1",
+    "@testing-library/dom": "10.4.1",
+    "@testing-library/jest-dom": "6.9.1",
+    "@testing-library/react": "16.3.0",
+    "@testing-library/user-event": "14.6.1",
+    "@types/babel__core": "7.20.5",
+    "@types/babel__preset-env": "7.10.0",
     "@types/buffer-crc32": "^0.2.4",
     "@types/bytebuffer": "^5.0.49",
     "@types/chai": "4.3.20",
     "@types/chai-as-promised": "^7.1.8",
     "@types/config": "3.3.5",
     "@types/dompurify": "^3.2.0",
     "@types/electron-localshortcut": "^3.1.3",
+    "@types/eslint": "~9.6.1",
+    "@types/eslint-plugin-mocha": "~10.4.0",
+    "@types/events": "~3.0.3",
     "@types/filesize": "5.0.2",
     "@types/firstline": "^2.0.4",
     "@types/fs-extra": "11.0.4",
     "@types/jsdom": "^21.1.7",
+    "@types/jsdom-global": "~3.0.7",
     "@types/libsodium-wrappers-sumo": "^0.7.8",
     "@types/linkify-it": "^5.0.0",
     "@types/lodash": "^4.17.16",
     "@types/mocha": "5.2.7",
     "@types/node-fetch": "^2.5.7",
-    "@types/react": "18.3.18",
-    "@types/react-dom": "18.3.5",
-    "@types/react-redux": "^7.1.33",
-    "@types/react-virtualized": "^9.21.30",
+    "@types/punycode": "~2.1.4",
+    "@types/react": "^19.2.7",
+    "@types/react-dom": "^19.2.3",
+    "@types/react-virtualized": "~9.22.3",
     "@types/rimraf": "4.0.5",
+    "@types/sass-loader": "~8.0.10",
     "@types/semver": "7.5.8",
     "@types/sinon": "9.0.11",
-    "@types/styled-components": "^5.1.34",
     "@typescript-eslint/eslint-plugin": "7.18.0",
     "@typescript-eslint/parser": "8.26.1",
+    "babel-plugin-react-compiler": "^1.0.0",
     "chai": "^4.5.0",
     "chai-as-promised": "^7.1.2",
     "chai-bytes": "^0.1.2",
     "concurrently": "^9.2.0",
     "cross-env": "^7.0.3",
     "css-loader": "^7.1.2",
+    "csstype": "^3.2.3",
     "electron": "34.2.0",
-    "electron-builder": "26.0.0-alpha.8",
+    "electron-builder": "26.0.12",
```

Comment:
this might break app packaging. We  picked the one that signal was using back in the day , and I think that was because more recent version were having an issue on one of the platforms